src/github.com/getlantern/lantern-ui/sass/_modals.sass

```diff
@@ -129,7 +129,8 @@ body[dir=rtl] .modal-footer
 #proxiedSitesLabelAndSearchContainer
   margin-top: $baseFontSize * 2
 
-#proxiedSitesSearch // override bootstrap
+// override bootstrap
```

Comment:
compass will report strange error if the comment follows selector, so I move it to separate line.